### ISSUE TITLE
Enable RethinkDB as an database alternative 

### ISSUE THREAD

**Issue by tsvetant** [AHAAAAAAA/PokemonGo-Map#2223]
It's great for realtime applications has an official python driver.
## Steps to Reproduce (for bugs)

1.
2.
3.
4.

## Your Environment

* Version used:
* Environment name and version (e.g. Python 2.7):
* Operating System and version (desktop or mobile):

**Comment by lisandro52** [comment]
Check @fourbytes 's fork. It supports rethinkdb, but is probably lacking other features from this repo.

**Comment by fourbytes** [comment]
Yep, my fork supports rethinkdb and heatmaps, and it should be mostly up to date with this repo (only a couple of commits behind). Only downside is it doesn't support SQL at all, but then again if you're making this issue then you probably don't care about that anyway. 😛
